src/System.Device.Gpio/System/Device/Gpio/GpioDriver.cs

```diff
@@ -9,17 +9,64 @@ namespace System.Device.Gpio
 {
     public abstract class GpioDriver : IDisposable
     {
+        [Obsolete("Available for backwards compatibility only")]
+        protected GpioDriver()
+        {
+            Board = null;
+        }
+
+        protected GpioDriver(Board board)
+        {
+            Board = board;
+        }
+
+        protected Board Board
+        {
+            get;
+        }
+
         /// <summary>
         /// The number of pins provided by the driver.
         /// </summary>
         protected internal abstract int PinCount { get; }
 
+        /// <summary>
+        /// Explicitly initializes the driver (instead of waiting for the first pin to be opened)
+        /// </summary>
+        public virtual void Initialize()
```

Comment:
Should this be abstract if it is empty?

Reply:
Basically yes, but then it would be breaking.